.github/workflows/web-deploy.yml

```diff
@@ -1,9 +1,8 @@
-name: deploy Capa Explorer Web to Github Pages
+name: deploy web to GitHub Pages
 
 on:
-  # Runs on pushes targeting the webui branch
   push:
-    branches: [ master ]
+    branches: [ master, "wb/webui-actions-1" ]
```

Comment:
this is the branch on my fork that i develop patches. since its not found under mandiant/capa, can we keep this around?